p2p/net/conn/listen.go

```diff
@@ -20,6 +22,26 @@ import (
 const SecioTag = "/secio/1.0.0"
 const NoEncryptionTag = "/plaintext/1.0.0"
 
+const connAcceptBuffer = 32
+const NegotiateReadTimeout = time.Second * 20
```

Comment:
in bad latency scenarios, a TCP handshake may very well take longer than 60 seconds.